File: src/main/java/com/clinica/atendimento/service/deepseek/dto/DeepSeekRequest.java
package com.clinica.atendimento.service.deepseek.dto;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class DeepSeekRequest {
    private String model;
    private String prompt;
    private boolean stream;

    public DeepSeekRequest(String model) {
        this.model = model;
        stream = false;
    }

    public DeepSeekRequest(String model, String prompt) {
        this(model);
        this.prompt = prompt;
    }
}

File: src/main/kotlin/com/clinica/atendimento/service/deepseek/dto/DeepSeekRequest.kt
package com.clinica.atendimento.service.deepseek.dto

data class DeepSeekRequest(
    var model: String = "deepseek-llm:latest",
    var prompt: String? = null,
    var stream: Boolean = false
) {
    constructor(prompt: String) : this() {
        this.prompt = prompt
    }
}